.github/workflows/translate.yml

```diff
@@ -22,6 +22,9 @@ jobs:
           submodules: recursive
           ref: 2.8
 
+      - name: l10n-clean
+        run: bash .github/workflows/run-in-docker.sh rabits/qt:5.12-desktop "cd translations && rm *.ts"
```

Comment:
Should be integrated into the Makefile. Also, we already have a `clean` job, so there's a name clash. The Makefile also contains a `clean` target already.